sys-cluster/teleport/files/teleport.confd

```diff
@@ -0,0 +1,30 @@
+# /etc/conf.d/teleport: config file for /etc/init.d/teleport
+
+# Where is your teleport.yaml file stored?
+
```

Comment:
I took the formatting from /etc/conf.d/sshd, seemed like a good idea to follow it's example.  Cleaned up the whitespace in: 40b03efc13c6b5de22388db877e7cf5a6429e649